pkg/controllers/managementuser/secret/secret.go

```diff
@@ -42,7 +58,23 @@ type Controller struct {
 	clusterName               string
 }
 
-func Register(ctx context.Context, cluster *config.UserContext) {
+type DirectSyncController struct {
```

Comment:
Is it resource sync of direct sync? I might just call it resource sync for clarity.